sw/device/sca/lib/sca.c

```diff
@@ -109,7 +115,7 @@ static void sca_init_gpio(sca_trigger_source_t trigger) {
 
   uint32_t select_mask =
       bitfield_field32_write(0, kTriggerSourceBitfield, UINT32_MAX);
-  uint32_t enable_mask = bitfield_bit32_write(0, kTriggerGateBitIndex, true);
+  uint32_t enable_mask = bitfield_bit32_write(0, kTriggerHwGateBitIndex, true);
```

Comment:
Right now, we always use the precise hardware-generated trigger. I think in the short term we should add simple_serial command to change this at runtime. I will work on this on THU. Ah, and I also need to test the CW305 and update kmac/otbn_serial etc.